### PR TITLE
Task 2 Completed

### LINKED ISSUE #3: Task 2 : Organising files in a directory
Monty has so many files in a folder with different extensions.He is facing trouble in accessing those files. He wants to organise all those files into different subfolders with names as per their extensions. For example, if a file has extension `.pdf` it should be in the folder PDF. Can you help him to do this task using python script ?

> Instructions

Download My Files folder and complete the task by strictly following the guidelines mentioned in the README.md file. Create PRs in the prescribed format only.

You can ask doubts if any, on the Slack Channel.

Prefer not to DM the mentors.

**NOTE:** The issue number of this task is 3 not 2.

### PR DESCRIPTION
**Pull request by uglyprincess**
__3__ 

3.
Fixes #3 

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

**Comment by uglyprincess** [comment]
folders

**Comment by uglyprincess** [comment]
doc

File: Submissions/uglyprincess/task_2/task2.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# path = "C:/Users/Aaryan/Desktop/Inconspicuous Folder/IT/Python/lol"
+
+# try:
+# 	os.mkdir(path)
+# except OSError:
+# 	print("Creation of the directory %s failed." %path)
+# else:
+# 	print("Successfully created the directory %s." %path)
+
+entries = os.listdir('../../../Task_2/My_Files')
```
Comment (Lokesh2703) [inline Submissions/uglyprincess/task_2/task2.py]: Actually u have to use os.path.join() to join path.

**Comment by uglyprincess** [comment]
Sir, I've used os.path.join() now.

**Comment by Lokesh2703** [comment]
U wont get points for mail code as this PR is for task 2

**Comment by Lokesh2703** [comment]
Dont submit organised folders in PR only submit the code.

**Comment by uglyprincess** [comment]
Okay sir

**Comment by uglyprincess** [comment]
Sir in the latest commit of this branch, I have not submitted the organized folders.

**Comment by uglyprincess** [comment]
Also sir, I have used a different branch of the forked repo for task 3

**Pull request merged**